libbeat/scripts/Makefile

```diff
@@ -203,6 +203,16 @@ coverage-report:
 update:
 	bash ${ES_BEATS}/libbeat/scripts/update.sh ${BEATNAME} ${BEAT_DIR}/${BEATNAME} ${ES_BEATS}/libbeat
 
+### KIBANA FILES HANDLING ###
+
+KIBANA_ES?=http://localhost:9200/
+.PHONY: update
+kibana-export:
+	python ${ES_BEATS}/libbeat/scripts/kibana_export.py --url ${KIBANA_ES} --dir $(shell pwd)/etc/kibana
+
+.PHONY: update
+kibana-import:
```

Comment:
@ruflin Do we want to have python as another dependency for all the Beats? It was a discussion around it [here](https://github.com/elastic/beats-dashboards/pull/85).